src/amplitude-client.js

```diff
@@ -283,7 +283,10 @@ AmplitudeClient.prototype._getInitialDeviceId = function (configDeviceId, stored
   }
 
   if (this.options.deviceIdFromUrlParam) {
-    return this._getDeviceIdFromUrlParam(this._getUrlParams());
+    let deviceIdFromUrlParam = this._getDeviceIdFromUrlParam(this._getUrlParams());
+    if(deviceIdFromUrlParam) {
```

Comment:
Could you add a space after the `if`. We'll try to add a better linting experience in the future 😩

Reply:
@jooohhn 
Thank you fore your review!
Sure. I'll fix it soon👍🏻